src/Inventory/Inventory.cpp

```diff
@@ -26,7 +26,7 @@ namespace CC
 		return elemental_[index];
 	}
 
-	void Inventory::useSpell(Spell spell)
+	void Inventory::useSpell(Spell &spell)
```

Comment:
If you don't intend to modify spell, I would recommend to add a const to your ref so that if you end up with a const spell and you need to use the spell, you won't be able.

Reply:
Quite right